understanding/21/pointer-gestures.html

```diff
@@ -12,24 +12,34 @@ <h1><strong>Pointer Gestures</strong><br />Understanding SC 2.5.1</h1>
 			<h2>Intent of this Success Criterion</h2>
 			<p>The intent of this Success Criterion is to ensure that content can be controlled with a range of pointing devices, abilities, and assistive technologies. Some people  cannot perform gestures in a precise manner, or they may use a specialized or adapted input device such as a head pointer, eye-gaze system, or speech-controlled mouse emulator. Some pointing methods lack the capability or accuracy to perform multipoint or path-based gestures.</p>
 
-			<p>A <strong>path-based gesture</strong> involves an interaction where not just the endpoints matter. If going  through an intermediate point (usually near the start of the gesture) also affects its meaning then it is a path-based gesture. The user engages a pointer (starting point), carries out a movement that goes through at least one intermediate-point before disengaging the pointer (end point). The intermediate point defines the gesture as requiring a specific path, even if the complete path is not defined.</p>
+			<p>A <strong>path-based gesture</strong> involves an interaction where not just the endpoints matter, but how the pointer moves between these points.</p>
+			
+			<p>If the gesture is only recognised if the user moves in a (mostly) straight line from the start point to the end point, it is an example of a <strong>path-based gesture</strong>.</p>
 
 			<figure id="figure-path-based-gesture-1">
-				<img src="img/path-based-gesture-1.png" width="400" alt="Hand showing a starting touch, 1. Moving through a second point, 2. Going to one of several points,3." />
-				<figcaption>A path-based gesture involves starting a pointer movement that goes through at least one intermediate point before the end-point. The end-point may be a continuation, or allow for various movements.</figcaption>
+				<img src="img/path-based-gesture-1.png" width="400" alt="Hand showing a starting touch, 1. Moving in a straight line to a second point, 2." />
+				<figcaption>A <strong>path-based gesture</strong> where pointer movement is only allowed in a straight line from the start-point to the end-point. If the user strays from the straight directional path, the gesture is not recognised, has no effect, or is aborted.</figcaption>
```

Comment:
The trouble is this still does not differentiate clearly between dragging movements and pointer gestures - because sliders on mobile may also require (initial) directional movement.
These can be moved in both ways so that andicates another differentiator - directionality. As said elsewhere, I think there are benefits in moving aways form the "intermediate point" definition because I have seen it cause lots of confusion. My alternative suggestion has been to define it in another way (DRAFT): 
"A pathbased gesture requires a particular direction of movement of the pointer across the display. It may in addtion require a particular speed or length of path for attached functions to be triggered"

That would differentiate better from axis-constrained dragging movements, because in these, their can be two directions of movement (as in an image slider).

Not that it matters that much, because 2.5.1 and 2.5.7 now both require a single pointer alternative.

Reply:
If a slider can be moved both ways, I think that's an example of two path-based-gestures (PBG), one for the first direction, another for the second direction.
Overall, we could consider the slider as either a PBG or dragging depending on the implementation, that's why it doesn't make a good example. 
I think this example description is ok though, because most sliders wouldn't fit into the "If the user strays from the straight directional path, the gesture is not recognised, has no effect, or is aborted."

Regarding the alternative description:
> A pathbased gesture requires a particular direction of movement of the pointer across the display.

Didn't we have examples where, so long as you started in a direction, it didn't require you to finish in that direction? That's why we used the intermediate point.
Also, what about multi-direction gestuers, e.g. up and left.